dbms/src/Storages/Page/PageStorage.h

```diff
@@ -233,7 +233,7 @@ class PageStorage : private boost::noncopyable
 
     virtual void drop() = 0;
 
-    virtual PageId getMaxId(NamespaceId ns_id) = 0;
+    virtual PageId getMaxId() = 0;
```

Comment:
How about renaming this method to `getEverMaxId()`? It means to return the max `id` this PS has ever met. And please add some documents.

Reply:
But in V2, this interface still keeps the old logic. How about just adding some notes about this interface?

Reply:
OK. Then please add detailed notes to the interface.

Reply:
done